docs/maui/essentials/badge.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Badge - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: The Badge allows developers to set the app icon badge number on the homescreen.
+ms.date: 07/14/2023
+---
+
+# Badge
+
+The `Badge` allows developers to set the app icon badge number on the homescreen.
+
+![Screenshot of an Badge on Windows](../images/essentials/badge-windows.png "Badge on Windows")
+
+---
+
+The following preconditions required for the `Badge`:
+
+# [iOS/MacCatalyst](#tab/ios)
+
+Authorize Badge permission for the app. It is automatically requested.
+
+# [Tizen](#tab/tizen)
+
+Add permissions to `tizen-manifest.xml`:
+
+```xml
+<privilege>http://tizen.org/privilege/notification</privilege>
+```
+
+## Syntax
+
+### C#
+
+The `Badge` can be used as follows in C#:
+
+```csharp
+void SetCount(uint value)
+{
+    Badge.Default.SetCount(value);
+}
+```
+
+## Methods
+
+|Method  |Description  |
+|---------|---------|
+| SetCount | Set the badge count. |
+
+## Dependency Registration
+
+In case you want to inject service, you first need to register it.
+Update `MauiProgram.cs` with the next changes:
+
+```csharp
+public static class MauiProgram
+{
+    public static MauiApp CreateMauiApp()
+    {
+        var builder = MauiApp.CreateBuilder();
+        builder
+            .UseMauiApp<App>()
+			.UseMauiCommunityToolkit();
+
+		builder.Services.AddSingleton<IBadge>(Badge.Default);
+        return builder.Build();
+    }
+}
+```
+
+Now you can inject the service like this:
+
+```csharp
+public partial class MainPage : ContentPage
+{
+    private readonly IBadge badge;
+
+	public MainPage(IBadge badge)
+	{
+		InitializeComponent();
+        this.badge = badge;
+	}
+	
+	public void SetCount(uint value)
+    {
+        badge.SetCount(value);
+    }
+}
+```
+
+## Examples
+
+You can find an example of `Badge` in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Essentials/BadgePage.xaml).
+
+## API
+
+You can find the source code for `Badge` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.Core/Essentials/Badge/IBadge.shared.cs).
+
+## Android
+
```

Comment:
I wonder if we can have a big !WARNING disclaimer making it clear that Android may not behave as expected?